src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -60,17 +60,14 @@
 	'memory_usage' => array('itemtitle' => gettext('Memory Usage'), 'category' => 'resource_usage', 'itemsort' => 19),
 	'swap_usage' => array('itemtitle' => gettext('Swap Usage'), 'category' => 'resource_usage', 'itemsort' => 20),
 	'disk_usage' => array('itemtitle' => gettext('Disk Usage'), 'category' => 'resource_usage', 'itemsort' => 21),
-	'admin_access_methods' => array('itemtitle' => gettext('Admin Lockdown'), 'category' => 'security', 'itemsort' => 22),
-	'admin_users' => array('itemtitle' => gettext('Users with Admin Access'), 'category' => 'security', 'itemsort' => 23)
 	);
 
 $validNames = array_keys($sysinfo_items);
 
-
+if (!function_exists('get_sysinfo_item_html')) {
 // get HTML for a sysinfo item
 
-
-function get_item_html($itemkey) {
+function get_sysinfo_item_html($itemkey) {
```

Comment:
I renamed the function because it seemed like other places could have a get_item_html() and making the name even more specific might save a hassle some day.